### DISCUSSION TITLE
打开之后显示以下文字后退出

### DISCUSSION THREAD

**Discussion by Lucky7Y** [mikumifa/biliTickerBuy#429] [提问专区]
![Snipaste_2024-09-25_14-56-50](https://github.com/user-attachments/assets/d2493028-356c-4ce5-8bc4-dfe57dc80df1)
![Snipaste_2024-09-25_14-59-14](https://github.com/user-attachments/assets/cdb61f13-e159-48d8-9be4-83d6f825166f)

**Comment by mikumifa** [comment]
关梯子
